core-services/image-mirroring/knative/OWNERS

```diff
@@ -2,4 +2,5 @@ approvers:
 - markusthoemmes
 - vdemeester
 - matzew
-
+- rhuss
+- navidshaikh
```

Comment:
I think we could remove the new owners for now (until they are part of openshift org) and get this PR going while we raise another one after the mentioned owners made it to openshift org.